packages/@react-aria/select/src/HiddenSelect.tsx

```diff
@@ -103,6 +103,7 @@ export function HiddenSelect<T>(props: HiddenSelectProps<T>) {
         <label>
           {label}
           <select {...selectProps}>
+            <option hidden disabled />
```

Comment:
Whats the reason for the `hidden` and `disabled` here?

Reply:
Oh, I forgot to delete them. 
Thank you!